derive/Cargo.toml

```diff
@@ -18,6 +18,6 @@ edition = "2021"
 proc-macro = true
 
 [dependencies]
-syn = "1.0"
+syn = { version = "2.0", default-features = false, features = ["printing", "parsing", "derive", "proc-macro"] }
```

Comment:
It seems according to 
https://docs.rs/syn/latest/syn/index.html#optional-features

Listing these 4 will at least avoid

> clone-impls (enabled by default) — Clone impls for all syntax tree types.